### PR TITLE
Development stellar helpers

### PR DESCRIPTION
**Pull request by waleedhammam**
- Change place sell order to manage sell order to have the ability to delete current order
- add a method to get the current placed orders

### CODE REVIEW

File: jumpscale/clients/stellar/stellar.py [hunk 1]
```diff
@@ -835,22 +836,24 @@ def get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
             asset_issuer = _NETWORK_KNOWN_TRUSTS[network].get(code, None)
             return Asset(code, asset_issuer)
 
-    def place_sell_order(
+    def manage_sell_order(
```
Comment (xmonader) [inline jumpscale/clients/stellar/stellar.py:839]: Let's create more helpers 
e.g (cancel_sell_order or delete_sell_order), better imo to pass different amount as flag for deletion

Comment (waleedhammam) [reply jumpscale/clients/stellar/stellar.py]: Done and testd, place_sell_order and cancel_sell_order

File: jumpscale/clients/stellar/stellar.py [hunk 2]
```diff
@@ -892,6 +911,8 @@ def manage_sell_order(
                 raise RuntimeError(f"couldn't sumbit transaction, probably unfunded") from e
```
Comment (xmonader) [inline jumpscale/clients/stellar/stellar.py:911]: typo should be submit instead of sumbit

Comment (waleedhammam) [reply jumpscale/clients/stellar/stellar.py:911]: actually, this whole message should be refactored, cause it only shows underfunded transaction, we can occur more errors like offer not found or whatever

**Pull request merged**